sdcm/nemesis.py

```diff
@@ -1775,29 +1775,31 @@ def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
     def _mgmt_backup(self, backup_specific_tables):
-        # TODO: When cloud backup is supported - add this to the below 'if' statement:
-        #  and not self.cluster.params.get('use_cloud_manager')
-        if not self.cluster.params.get('use_mgmt'):
-            raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
-        if not self.cluster.params.get('backup_bucket_location'):
-            raise UnsupportedNemesis('backup bucket location configuration is not defined!')
-
         mgr_cluster = self.cluster.get_cluster_manager()
-        cluster_backend = self.cluster.params.get('cluster_backend')
-        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
-        if self._is_it_on_kubernetes():
-            bucket_name = f"s3:{backup_bucket_location.split()[0]}"
-        elif cluster_backend == 'aws':
-            bucket_name = f"s3:{backup_bucket_location.split()[0]}"
-        elif cluster_backend == 'gce':
-            bucket_name = f"gcs:{backup_bucket_location}"
+        if self.cluster.params.get('use_cloud_manager'):
+            auto_backup_task = mgr_cluster.backup_task_list[0]
+            location = auto_backup_task.arguments.split('-L')[1].split()[0]
         else:
-            raise ValueError(f"cluster_backend={cluster_backend} not supported in ManagementBackup")
+            if not self.cluster.params.get('use_mgmt'):
+                raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
+            if not self.cluster.params.get('backup_bucket_location'):
+                raise UnsupportedNemesis('backup bucket location configuration is not defined!')
+
+            cluster_backend = self.cluster.params.get('cluster_backend')
+            backup_bucket_location = self.cluster.params.get('backup_bucket_location')
+            if self._is_it_on_kubernetes():
+                location = f"s3:{backup_bucket_location.split()[0]}"
+            elif cluster_backend == 'aws':
+                location = f"s3:{backup_bucket_location.split()[0]}"
+            elif cluster_backend == 'gce':
+                location = f"gcs:{backup_bucket_location}"
+            else:
+                raise ValueError(f"cluster_backend={cluster_backend} not supported in ManagementBackup")
         if backup_specific_tables:
             non_test_keyspaces = self.cluster.get_test_keyspaces()
```

Comment:
this is wrong... as `get_test_keyspaces()` returns `test_keyspaces`, so the naming must be changed (I know, it was already in there, but it is misleading)

Reply:
Not in the scope of this PR.